procrastinate/app.py

```diff
@@ -249,6 +249,10 @@ async def run_worker_async(self, **kwargs) -> None:
             Raising this parameter can lower the rate of workers making queries to the
             database for requesting jobs.
             (defaults to 5.0)
+        listen_notify : ``bool``
+            If ``True``, worker will dedicate a connection from the pool to listening
+            for database event, notifying of newly available jobs. If ``False``, workers
+            will just poll the database regularily (see ``timeout``). (defaults to True)
```

Comment:
s/regularily/regularly ?

Reply:
And you may want to use `periodically` here, which you use elsewhere. Up to you.